transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/PaymentInformation.java

```diff
@@ -6,23 +6,23 @@
 import pl.adamboguszewski.transaction.service.service.api.transaction.CreateTransactionRequest;
 
 @Value
-public class TransactionPayment {
+public class PaymentInformation {
 
     Long amountPaid;
     Long multiplier;
     Currency currency;
     PaymentType paymentType;
 
-    public TransactionPayment(Long amountPaid, Long multiplier, Currency currency,
+    public PaymentInformation(Long amountPaid, Long multiplier, Currency currency,
```

Comment:
Same thing, shouldn't it also be private?

Reply:
yes